tests/web/test_main.py

```diff
@@ -470,12 +470,38 @@ def test_get_environments(project_context: Context) -> None:
     response = client.get("/api/environments")
     assert response.status_code == 200
     response_json = response.json()
-    assert len(response_json) == 1
+    assert len(response_json["environments"]) == 1
 
-    environment = Environment.parse_obj(response_json["prod"])
+    environment = Environment.parse_obj(response_json["environments"]["prod"])
     assert environment == Environment(
         name="prod", snapshots=[], start_at="1970-01-01", plan_id="", suffix_target="schema"
     )
+    assert response_json["pinned_environments"] == list(project_context.config.pinned_environments)
+    assert (
+        response_json["default_target_environment"]
+        == project_context.config.default_target_environment
+    )
+
+
+async def test_delete_environment_success(project_context: Context):
```

Comment:
I think you have to mark asyncio tests for pytest to pick them up, but this doesn't need to be async. 

```suggestion
def test_delete_environment_success(project_context: Context):
```